### ISSUE TITLE
Fallout 4 Full screen

### ISSUE THREAD

**Issue by SaveAndrea** [Codeusa/Borderless-Gaming#412]
When I run the Borderless gaming app, it loads the game into full screen, but the resolution is far too large and it only shows about 25% of the screen.
![screenshot 2018-09-03 17 12 09](https://user-images.githubusercontent.com/42929957/44975125-256e5680-af51-11e8-9fcf-7377f2fe1c8a.png)
![screenshot 2018-09-03 17 12 24](https://user-images.githubusercontent.com/42929957/44975126-256e5680-af51-11e8-812b-2770a6fc9ef2.png)
![screenshot 2018-09-03 17 12 30](https://user-images.githubusercontent.com/42929957/44975127-256e5680-af51-11e8-8542-03ac42da7dda.png)

What should I do to fix this issue?

**Comment by benzittlau** [comment]
I see this same issue, resizing Fallout 4 in a borderless window of 2560x1440 up to 4k. One of the surprising things is in-game the hands still render scaled appropriately, but the game and UI don't appear to be.

## Scene without scaling the borderless window:
![image](https://user-images.githubusercontent.com/366749/49066925-cc286300-f1df-11e8-8e8c-a7a3b2f5219d.png)


## With Borderless Gaming enabled:
![image](https://user-images.githubusercontent.com/366749/49066682-25dc5d80-f1df-11e8-95af-2bacd3cbc244.png)

Another interesting data point is that there's a setting in the game's preferences that will "Maximize" when in a windowed mode, `bMaximizeWindow=1`.  With this set, *without* borderless gaming, I see the same thing.  I also see the same effect with Fullscreenizer, and ShiftWindow.

Seems like it's something not specific to borderless gaming, but would love to understand what the isssue going on here is and what the possible solutions might be.

**Comment by benzittlau** [comment]
I found a fix for this, though I don't fully understand *why* the fix worked.  By right-clicking on the Fallout 4 executable, and changing this setting to for "Scaling performed by:" from "Application" to "System (Enhanced)", or by disabling the override:

![image](https://user-images.githubusercontent.com/366749/49105731-a5e5e000-f23e-11e8-9097-32f00119f5a5.png)

When I start Fallout 4 with that changed, it automatically starts as a full screen borderless window for my 4k display, running at 2560x1440 (my resolution set for the game).

I discovered this setting simply by poking around in the compatibility settings and finding something that sounded promising and testing it, so I really have no idea *why* this is working.  In particular what's odd/interesting is I'm not using *anything* to maximize the window (no 3rd party apps, and not Fallout 4's "maximize" setting), so I'm not even really sure why it's being maximized to full screen.

I might try and dig into this further to understand *why* this worked, but I also might just actually finally play some Fallout 4 instead ;)  If I *do* learn more I'll post back and share those findings here.
